packages/core/ios/Tests/DdSdkTests.swift

```diff
@@ -1108,6 +1108,36 @@ class DdSdkTests: XCTestCase {
 
         XCTAssertNotNil(core.baggages["browser-rum-event"])
     }
+    
+    func testInitialResourceThreshold() {
+        let configuration: DdSdkConfiguration = .mockAny(initialResourceThreshold: 0.5)
+        
+        let ddConfig = DdSdkNativeInitialization().buildRUMConfiguration(configuration: configuration, eventEmitter: nil)
+        
+        let predicate = ddConfig.networkSettledResourcePredicate
+        let earlyResource = TNSResourceParams(
+            url: "https://datadoghq.com",
+            timeSinceViewStart: 0.1,
+            viewName: "Home"
+        )
+        
+        let onThresholdLImitResource = TNSResourceParams(
```

Comment:
Small typo (capital 'I' in 'Limit'):

```suggestion
        let onThresholdLimitResource = TNSResourceParams(
```

Reply:
Nice catch! 🙇